XRTK-Core/Packages/com.xrtk.core/Utilities/ValidateConfiguration.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Text;
+using UnityEditor;
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.Extensions;
+using XRTK.Providers.Controllers;
+using XRTK.Services;
+
+namespace XRTK.Utilities.Editor
```

Comment:
Is this in a valid editor/inspector folder with an assembly definition?

Reply:
Namespace doesn't match the file location

Reply:
removed editor namespace